subprojects/packagefiles/tracy-0.8/meson.build

```diff
@@ -0,0 +1,78 @@
+project('tracy', ['cpp'], version : '0.8')
+
+add_project_arguments('-DTRACY_EXPORTS', language : 'cpp')
+add_project_arguments('-DTRACY_ENABLE', language : 'cpp')
+add_project_arguments('-DTRACY_ONLY_LOCALHOST', language : 'cpp')
+add_project_arguments('-DTRACY_ON_DEMAND', language : 'cpp')
+add_project_arguments('-DTRACY_NO_CALLSTACK', language : 'cpp')
+add_project_arguments('-DTRACY_DELAYED_INIT', language : 'cpp')
+
+threads_dep = dependency('threads')
```

Comment:
Is all of this needed? I thought we're just adding tracing for vkd3d-proton here.

Reply:
The no_callstack and delayed_init were added to make Tracy work. The thread_local initialization and usage within Tracy was having difficulties on my machine.